script.module.pysubs2/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="script.module.pysubs2" name="pysubs2 module" version="1.5.0" provider-name="Tomáš Karabela">
+<addon id="script.module.pysubs2" name="pysubs2 module" version="1.5.1" provider-name="Tomáš Karabela">
```

Comment:
there's no 1.5.1 version on the upstream repo. I suggest you add a version identifier instead:
1.5.0+matrix.1